<launch>
  <env name="PYTHONPATH" value="$(env PYTHONPATH)" />

  <node pkg="nao_accuracy_challenge" type="nao_speech.py" name="nao_speech_service" required="true" output="screen"/>
  <node pkg="nao_accuracy_challenge" type="shooting_service.py" name="ShootBow" required="true" output="screen"/>
  <node pkg="nao_accuracy_challenge" type="vision_service.py" name="NAO_VISION" required="true" output="screen"/>
  <node pkg="nao_accuracy_challenge" type="control_service.py" name="NAO_CONTROL" required="true" output="screen"/>
  <node pkg="nao_accuracy_challenge" type="nao_tactile.py" name="nao_tactile_touch" required="true" output="screen"/>
  <node pkg="nao_accuracy_challenge" type="nao_leds.py" name="nao_leds" required="true" output="screen"/>

</launch>

